index.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express')
+const { MongoClient } = require('mongodb')
+const bodyParser = require('body-parser')
+const jsonParser = bodyParser.json()
+
+MongoClient.connect('mongodb://localhost/notebook', (err, db) => {
+
+  if (err) {
+    console.error(err)
+    process.exit(1)
+  }
+
+  const notes = db.collection('notes')
+  const app = express()
+  app.use(jsonParser)
+
+  app.post('/api/notes', (req, res) => {
+    notes
+      .save(req.body, (err, result) => {
```

Comment:
`save` is deprecated. Favor `insertOne`.

Reply:
Updated.